charts/__tests__/gardener-dashboard.spec.js

```diff
@@ -516,6 +516,71 @@ describe('gardener-dashboard', function () {
           })
         })
       })
+      describe('themes', function () {
+        it('should render the template', async function () {
+          // eslint-disable-next-line no-unused-vars
+          const values = writeValues(filename, {
+            frontendConfig: {
+              themes: {
+                light: {
+                  primary: '#ff0000',
+                  'main-navigation-title': 'colors.grey.darken3'
```

Comment:
```js
                  'main-navigation-title': 'grey.darken3'
```
as discussed previously also with @holgerkoser out of band, specify only the path

Reply:
done